app/src/main/java/org/simple/clinic/analytics/TimingCollector.kt

```diff
@@ -0,0 +1,98 @@
+package org.simple.clinic.analytics
```

Comment:
We could reword this commit. Its not very informative.
How about `Add a TimingCollector to report time taken for an operation`?

Reply:
Done.